Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
```

Comment:
That being said, the purpose of this router is to get the schema and model in placed. I presume that routing and business logic will be a later addition.